File: tl/src/main/java/com/github/badoualy/telegram/tl/api/auth/TLAbsSentCodeType.java
package com.github.badoualy.telegram.tl.api.auth;

import com.github.badoualy.telegram.tl.core.TLObject;

/**
 * Abstraction level for the following constructors:
 * <ul>
 * <li>{@link TLSentCodeTypeApp}: auth.sentCodeTypeApp#3dbb5986</li>
 * <li>{@link TLSentCodeTypeCall}: auth.sentCodeTypeCall#5353e5a7</li>
 * <li>{@link TLSentCodeTypeFlashCall}: auth.sentCodeTypeFlashCall#ab03c6d9</li>
 * <li>{@link TLSentCodeTypeSms}: auth.sentCodeTypeSms#c000bba2</li>
 * </ul>
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
public abstract class TLAbsSentCodeType extends TLObject {

    public TLAbsSentCodeType() {
    }
}

File: tl/src/main/kotlin/com/github/badoualy/telegram/tl/api/auth/TLAbsSentCodeType.kt
package com.github.badoualy.telegram.tl.api.auth

import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_BOOLEAN
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_CONSTRUCTOR_ID
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_DOUBLE
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_INT32
import com.github.badoualy.telegram.tl.TLObjectUtils.SIZE_INT64
import com.github.badoualy.telegram.tl.TLObjectUtils.computeTLBytesSerializedSize
import com.github.badoualy.telegram.tl.TLObjectUtils.computeTLStringSerializedSize
import com.github.badoualy.telegram.tl.core.TLObject

/**
 * Abstraction level for the following constructors:
 * * [auth.sentCodeTypeApp#3dbb5986][TLSentCodeTypeApp]
 * * [auth.sentCodeTypeCall#5353e5a7][TLSentCodeTypeCall]
 * * [auth.sentCodeTypeFlashCall#ab03c6d9][TLSentCodeTypeFlashCall]
 * * [auth.sentCodeTypeSms#c000bba2][TLSentCodeTypeSms]
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
abstract class TLAbsSentCodeType : TLObject()
